.github/workflows/main.yml

```diff
@@ -0,0 +1,45 @@
+name: Continuous Integration
+
+on: [pull_request]
+
+env:
+  RUBY_VERSION: 2.7
+  RUBOCOP_VERSION: 0.90.0
+
+jobs:
+  rubocop:
+    name: runner / rubocop
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: reviewdog/action-rubocop@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-check
+          rubocop_version: ${{ env.RUBOCOP_VERSION }}
```

Comment:
Instead of relying on an external action here, can we just do it ourselves as we are with rspec, so we're in control of what's happening?